### PR TITLE
Remove PPE endpoints in samples

### PR DESCRIPTION
**Pull request by derisen**
Remove PPE endpoints and hardcoded IDs as defaults in samples to encourage customers for registering their own apps and prevent leakage.

* Clear all default configs in msal-node samples
* Add environments for test, prod to msal-angular-v2, msal-react samples
* Refactor msal-browser e2e tests to use a single sample, clear the rest

p.s. apologies for the large PR size, can possibly break it down to 3

### CODE REVIEW

File: samples/msal-angular-v2-samples/angular10-sample-app/src/environments/environment.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export const environment = {
+  production: false,
+  msalConfig: {
+    auth: {
+      clientId: '3fba556e-5d4a-48e3-8e1a-fd57c12cb82e', // PPE testing environment
```
Comment (peterzenz) [inline samples/msal-angular-v2-samples/angular10-sample-app/src/environments/environment.e2e.ts:5]: Are we not removing PPE and clientIds?  Or is this for just for our e2e tests?

File: samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/browserAAD.spec.ts [hunk 2]
```diff
@@ -165,7 +163,7 @@ describe("AAD-PPE Tests", () => {
         beforeEach(async () => {
             context = await browser.createIncognitoBrowserContext();
             page = await context.newPage();
-            page.setDefaultTimeout(ONE_SECOND_IN_MS*5);
+            page.setDefaultTimeout(ONE_SECOND_IN_MS*10);
```
Comment (peterzenz) [inline samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/browserAAD.spec.ts:166]: Reliability issues on these?  You're making this change everywhere.

File: samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/clientCapabilities.spec.ts [hunk 3]
```diff
@@ -35,12 +40,17 @@ describe("Browser tests", function () {
     beforeEach(async () => {
         context = await browser.createIncognitoBrowserContext();
         page = await context.newPage();
-        page.setDefaultTimeout(ONE_SECOND_IN_MS*5);
-        BrowserCache = new BrowserCacheUtils(page, "sessionStorage");
+        page.setDefaultTimeout(ONE_SECOND_IN_MS * 5);
```
Comment (peterzenz) [inline samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/clientCapabilities.spec.ts:43]: Except here, keeping this one at 5?  On line 136 as well.

**Review by peterzenz — APPROVED** [review]

File: samples/msal-angular-v2-samples/angular10-sample-app/src/environments/environment.e2e.ts (hunk 1 above)
Comment (derisen) [reply samples/msal-angular-v2-samples/angular10-sample-app/src/environments/environment.e2e.ts:5]: At the moment we have keep them for E2E tests, but checking if we can fetch these at runtime from the lab -if so we can remove them completely

File: samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/browserAAD.spec.ts (hunk 2 above)
Comment (derisen) [reply samples/msal-browser-samples/VanillaJSTestApp2.0/app/customizable-e2e-test/test/browserAAD.spec.ts:166]: These were consistently timing out on local. There's a 30000 ms cap for tests defined as default [here ](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/09f65ee73fa849e71e3a73b857f562fab9d47263/samples/msal-browser-samples/VanillaJSTestApp2.0/jest.config.js#L14). Need to consult why this was defined on the individual test level.

**Comment by derisen** [comment]
Closing this as discussed. Will break it down to smaller PRs -thanks!

**Pull request closed without merging**